### PR TITLE
Update to include nuget-validation build.cake

### LINKED ISSUE #723: Add Nuget validation to the build.cake

### PR DESCRIPTION
**Pull request by newky2k**
fixes #723

### CODE REVIEW

**Comment by mikekinsman** [comment]
OPS Build status updates of commit _[101765e](https://github.com/mono/SkiaSharp/pull/724/commits/101765ed5cfccf21e96dffee4ca2198afe007b88)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[build.cake](https://github.com/mono/SkiaSharp/blob/dh_NugetValidation/build.cake) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C10%5C81ca5486-8c50-0a94-05c4-486e23e99ecf%5CPullRequest%5C201812102042426370-724%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=edwxlm0zMnzczHhU%2FvCl04Xq4WMh0xQ90TtRdsmxMes%3D&st=2018-12-10T20%3A44%3A22Z&se=2019-01-10T20%3A49%3A22Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

File: build.cake [hunk 1]
```diff
@@ -398,6 +399,47 @@ Task ("nuget-only")
     }
 });
 
+Task("nuget-validation")
+    .IsDependentOn("nuget")
+	.Does(()=>
```
Comment (mattleibow) [inline build.cake]:
```suggestion
    .Does(()=>
```

File: build.cake [hunk 2]
```diff
@@ -398,6 +399,47 @@ Task ("nuget-only")
     }
 });
 
+Task("nuget-validation")
+    .IsDependentOn("nuget")
+    .Does(()=>
+{
+	//setup validation options
```
Comment (mattleibow) [inline build.cake]:
```suggestion
	// setup validation options
```

**Comment by mikekinsman** [comment]
OPS Build status updates of commit _[5c8a044](https://github.com/mono/SkiaSharp/pull/724/commits/5c8a0445baa4ab59e90574b9f82d08be467e3ee9)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[build.cake](https://github.com/mono/SkiaSharp/blob/dh_NugetValidation/build.cake) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C14%5C81ca5486-8c50-0a94-05c4-486e23e99ecf%5CPullRequest%5C201812141858129859-724%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=wvbK4bSQ7Fs%2FdONXnkIoPmlyDx1eEy7IlhWSh2PYkbs%3D&st=2018-12-14T18%3A55%3A35Z&se=2019-01-14T19%3A00%3A35Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by mikekinsman** [comment]
OPS Build status updates of commit _[37e6173](https://github.com/mono/SkiaSharp/pull/724/commits/37e61730df4d979bbd1bd39270c2747f5c2c6966)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[build.cake](https://github.com/mono/SkiaSharp/blob/dh_NugetValidation/build.cake) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C14%5C81ca5486-8c50-0a94-05c4-486e23e99ecf%5CPullRequest%5C201812141859071083-724%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=1tu2PayWoBpmn7WsODP2WPh0U2y%2FsPf1gAaKPOKilpM%3D&st=2018-12-14T18%3A58%3A01Z&se=2019-01-14T19%3A03%3A01Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by mikekinsman** [comment]
OPS Build status updates of commit _[37e6173](https://github.com/mono/SkiaSharp/pull/724/commits/37e61730df4d979bbd1bd39270c2747f5c2c6966)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[build.cake](https://github.com/mono/SkiaSharp/blob/dh_NugetValidation/build.cake) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C14%5C81ca5486-8c50-0a94-05c4-486e23e99ecf%5CPullRequest%5C201812141858568783-724%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Z1i14FM6CthXj9fcx%2FmRbPmTbXygTRBnDPjQXRiHj60%3D&st=2018-12-14T19%3A00%3A26Z&se=2019-01-14T19%3A05%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by mikekinsman** [comment]
OPS Build status updates of commit _[87c809c](https://github.com/mono/SkiaSharp/pull/724/commits/87c809c527eedb298dc508b4a1e6c456fae95fbf)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[build.cake](https://github.com/mono/SkiaSharp/blob/dh_NugetValidation/build.cake) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C14%5C81ca5486-8c50-0a94-05c4-486e23e99ecf%5CPullRequest%5C201812141903268536-724%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=MB5q%2FF6anz8C3MTRoXRkZ3t9fPlTm94oCa%2F7Mt1dly4%3D&st=2018-12-14T19%3A03%3A08Z&se=2019-01-14T19%3A08%3A08Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Comment by mikekinsman** [comment]
OPS Build status updates of commit _[8dd2448](https://github.com/mono/SkiaSharp/pull/724/commits/8dd2448243328cb1aea76426a82f7a02ee836fb5)_: 

### :white_check_mark: Validation status: passed


File | Status | Preview URL | Details
---- | ------ | ----------- | -------
[build.cake](https://github.com/mono/SkiaSharp/blob/dh_NugetValidation/build.cake) | :white_check_mark:Succeeded | |

For more details, please refer to the [build report](https://opbuildstorageprod.blob.core.windows.net/report/2018%5C12%5C14%5C81ca5486-8c50-0a94-05c4-486e23e99ecf%5CPullRequest%5C201812141905356948-724%5Cworkflow_report.html?sv=2015-02-21&sr=b&sig=Gyyf6D2FlvHtG2NI6h0AWzvwkreAUuqzeTN7l4qkd8U%3D&st=2018-12-14T19%3A05%3A26Z&se=2019-01-14T19%3A10%3A26Z&sp=r).

**Note:** If you changed an existing file name or deleted a file, broken links in other files to the deleted or renamed file are listed only in the full build report.

**Pull request merged**